src/services/storage.cpp

```diff
@@ -170,6 +172,85 @@ void Storage::handleTrivialError(const leveldb::Status &status) {
   }
 }
 
+void Storage::saveBlock(Block &block) {
+  // Meta
+  BytesBuilder meta_header_builder;
+
+  string compression_algo_type_str;
+  if (block.compression_algo_type == CompressionAlgorithmType::LZ4)
+    compression_algo_type_str = "LZ4";
+  else
+    compression_algo_type_str = "NONE";
+
+  meta_header_builder.append(compression_algo_type_str);
```

Comment:
meta만 들어감

Reply:
```suggestion
  meta_header_builder.append(block.compression_algo_type, 1);
```
compression_algo_type이 uint8_t니까 이렇게 하면 됩니다. byte builder를 조금 더 쓰기쉽게 만들어야 겠네요. 정확히 쓰는 장면을 상상 못해서 죄송...